ddtrace/span.py

```diff
@@ -373,7 +373,7 @@ def get_tag(self, key):
         """Return the given tag or None if it doesn't exist."""
         return self._meta.get(key, None)
 
-    def _get_tags(self):
+    def get_tags(self):
         # type: () -> _MetaDictType
```

Comment:
I wonder if we shouldn't be exposing `_MetaDictType` publicly

Reply:
That's a good point and we could reconsider adding types to our public API.

For now, I don't think this is a problem as the autodoc output does not use this internal alias but instead displays the union type:

https://1046382-61572326-gh.circle-artifacts.com/0/tmp/docs/api.html#ddtrace.Span.get_tag

Reply:
Would it get exposed as `_MetaDictType` or the expanded version? 🤔 

As that name and nothing else, kind of useless 🤔